test/suites/container_devices_nic_bridged_filtering.sh

```diff
@@ -35,7 +35,7 @@ test_container_devices_nic_bridged_filtering() {
 
   lxc network set "${brName}" ipv6.dhcp.stateful true
   lxc network set "${brName}" bridge.hwaddr 00:11:22:33:44:55
-  lxc network set "${brName}" ipv4.address 192.0.2.1/24
+  lxc network set "${brName}" ipv4.address 192.0.2.1/16
```

Comment:
Why the /16 here, rather than using a discrete /24 for the additional routed subnets?

Reply:
In order to test whether the `ipv4.routes` are not blocked I need to assign the container an IP within that range and check connectivity. My thinking was that I wouldn't be able to assign an IP to the container that was outside the parent network. So the `ipv4.address`, `ipv4.routes`, and `ipv4.routes.external` would all need to be contained within the parent network.